Makefile

```diff
@@ -14,12 +14,19 @@ HANDLER_IMAGE ?= $(IMAGE_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)
 GINKGO_EXTRA_ARGS ?=
 GINKGO_ARGS ?= -v -r --randomizeAllSpecs --randomizeSuites --race --trace $(GINKGO_EXTRA_ARGS)
 GINKGO?= go run ./vendor/github.com/onsi/ginkgo/ginkgo
-
-KUBECONFIG ?= ./cluster/.kubeconfig
-# TODO: use operator-sdk from vendor/
 OPERATOR_SDK ?= go run ./vendor/github.com/operator-framework/operator-sdk/cmd/operator-sdk
 LOCAL_REGISTRY ?= registry:5000
-KUBECTL ?= ./cluster/kubectl.sh
+
+export KUBEVIRT_PROVIDER ?= k8s-1.13.3
```

Comment:
please check that our developer docs are up to date with this (especially the example of running OpenShift local cluster)

Reply:
Fixed, going back to 1.11.0 I was all wrong with that.